nat-lab/tests/test_cleanup.py

```diff
@@ -41,7 +41,7 @@ async def test_get_network_interface_tunnel_keys(adapter_type, name) -> None:
         # that might have managed to survive the end of the previous test.
         keys = await _get_network_interface_tunnel_keys(connection)
         assert [
-            "HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Control\\Class\\{4d36e972-e325-11ce-bfc1-08002be10318}\\0006"
+            "HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Control\\Class\\{4d36e972-e325-11ce-bfc1-08002be10318}\\0005"
```

Comment:
Please add explanation for this change in the commit message

Reply:
Updated commit message and added explanation in PR description